packages/viz/package.json

```diff
@@ -40,6 +40,7 @@
     "ts-loader": "^6.2.1",
     "typescript": "^3.6.3",
     "tslib": "^1.10.0",
+    "@danmarshall/deckgl-typings": "3.4.6",
```

Comment:
Cool, I didn't know about them.

I have checked the proj, and it looks like they will try to get up to v8 (but it looks like they don't have many contributors...). Not sure if in the meantime using 'old' typings (for v 7.x.x.) could cause any issue.

Reply:
Not right now, because we're only using it for this declaration:

https://github.com/CartoDB/toolkit/blob/19d3b65498e0c381d90176d01f0c1c2fcd73ac6a/packages/viz/src/declarations/deckgl.d.ts#L3-L9